pkg/tagger/collectors/workloadmeta_extract.go

```diff
@@ -231,6 +232,8 @@ func (c *WorkloadMetaCollector) handleContainerImage(ev workloadmeta.Event) []*T
 
 	tags := utils.NewTagList()
 	tags.AddLow("image_name", image.Name)
+	tags.AddLow("image_repo_tags", strings.Join(image.RepoTags, ","))
+	tags.AddLow("image_repo_digests", strings.Join(image.RepoDigests, ","))
```

Comment:
I don't think it is a good idea to add this information on every containers in the tagger, because the value is quite large and it will be added to any metrics, and we already have `image_tag`. 

Also a technical comment here. It is better to add several time the same tag key with different value instead of doing a values join.

  ```golang
	for _, value := range image.RepoTags {
	    tags.AddLow("image_repo_tags", value)
	}
	# ...
```

Reply:
Another issue, is that this tag might change during the lifecycle of the container, because if the same image is pull with another tag or registry on the same host. the value in the tag will be modified.

Reply:
Ok, I think we may do without the repoTags and repoDigests. But adding just the image_id is ok right ?